doobie/src/main/scala/tofu/doobie/transactor/Txr.scala

```diff
@@ -42,11 +42,16 @@ trait Txr[F[_]] {
 }
 
 object Txr {
```

Comment:
I think is better to have scaladoc here with a bit of motivation. Like what types one should use.